src/librustc_error_codes/error_codes/E0627.md

```diff
@@ -0,0 +1,30 @@
+A yield statement was used outside of the generator literal.
```

Comment:
Yield is an expression form, not a statement.

Reply:
@Centril Thanks for the hint. Fixed.